### PR TITLE
overTheBox: replace oui-loader by oui-spinner

### PR DESCRIPTION
**Pull request by virgotopia**

### CODE REVIEW

File: client/app/telecom/freeFax/faxConfiguration/freeFax-faxConfiguration.html [hunk 1]
```diff
@@ -3,9 +3,8 @@
         <h3 class="oui-header_3 oui-tile__title"
             data-translate="freefax_fax_configuration">
         </h3>
-        <div class="oui-tile__body"
-             data-ng-if="loading">
-            <oui-loader data-size="m"></oui-loader>
+        <div class="oui-tile__body" data-ng-if="loading">
+            <oui-spinner></oui-spinner>
```
Comment (antleblanc) [inline client/app/telecom/freeFax/faxConfiguration/freeFax-faxConfiguration.html]: missing text-center?

File: client/app/telecom/freeFax/information/freeFax-information.html [hunk 2]
```diff
@@ -2,9 +2,8 @@
     <h3 class="oui-header_3 oui-tile__title"
         data-translate="freefax_general_information">
     </h3>
-    <div class="oui-tile__body"
-         data-ng-if="loading">
-        <oui-loader data-size="m"></oui-loader>
+    <div class="oui-tile__body" data-ng-if="loading">
+        <oui-spinner></oui-spinner>
```
Comment (Jisay) [inline client/app/telecom/freeFax/information/freeFax-information.html]: missing `text-center` class

File: client/app/telecom/freeFax/voicemailConfiguration/freeFax-voicemailConfiguration.html [hunk 3]
```diff
@@ -179,10 +179,10 @@
                                 data-ng-disabled="VoicemailConf.loading"
                                 data-translate="cancel">
                             </button>
-                            <oui-loader class="ml-2"
-                                        data-ng-show="VoicemailConf.loading"
-                                        data-size="s">
-                            </oui-loader>
+                            <oui-spinner class="ml-2"
```
Comment (Jisay) [inline client/app/telecom/freeFax/voicemailConfiguration/freeFax-voicemailConfiguration.html]: should be centered

File: client/app/telecom/orders/telecom-orders.html [hunk 4]
```diff
@@ -57,7 +57,9 @@ <h1 data-translate="telecom_orders_follow_up_title"></h1>
                         </tr>
                         <tr data-ng-show="OrdersCtrl.loading">
                             <td colspan="5">
-                                <oui-loader data-size="m"></oui-loader>
+                                <div class="text-center">
```
Comment (Jisay) [inline client/app/telecom/orders/telecom-orders.html]: apply `text-center` class directly on the `td` element

File: client/app/telecom/freeFax/notifications/freeFax-notifications.html [hunk 5]
```diff
@@ -49,7 +49,9 @@
                     </tr>
                     <tr data-ng-hide="FreeFaxNotifications.notifications">
                         <td colspan="4">
-                            <oui-loader data-size="m"></oui-loader>
+                            <div class="text-center">
```
Comment (Jisay) [inline client/app/telecom/freeFax/notifications/freeFax-notifications.html]: apply `text-center` class directly on the `td` element

File: client/app/telecom/dashboard/bills/telecom-dashboard-bills.html [hunk 6]
```diff
@@ -72,7 +72,9 @@
             <tbody data-ng-if="BillsCtrl.loaders.bills">
                 <tr>
                     <td colspan="4">
-                        <oui-loader data-size="m"></oui-loader>
+                        <div class="text-center">
```
Comment (Jisay) [inline client/app/telecom/dashboard/bills/telecom-dashboard-bills.html]: apply `text-center` class directly on the `td` element

File: client/app/telecom/overTheBox/details/overTheBox-details.html [hunk 7]
```diff
@@ -222,7 +222,7 @@
                         </h3>
                         <div class="oui-tile__body"
```
Comment (Jisay) [inline client/app/telecom/overTheBox/details/overTheBox-details.html]: should center the spinner :-)

File: client/app/telecom/overTheBox/details/overTheBox-details.html [hunk 8]
```diff
@@ -256,7 +256,7 @@
...
```
Comment (Jisay) [inline client/app/telecom/overTheBox/details/overTheBox-details.html]: here too :-)

File: client/app/telecom/overTheBox/remote/overTheBox-remote.html [hunk 9]
```diff
@@ -27,7 +27,9 @@
                     <tbody data-ng-if="OverTheBoxRemote.remotes.length === 0">
                         <tr data-ng-show="OverTheBoxRemote.loading">
                             <td colspan="5">
-                                <oui-loader data-size="m"></oui-loader>
+                                <div class="text-center">
```
Comment (Jisay) [inline client/app/telecom/overTheBox/remote/overTheBox-remote.html]: apply `text-center` class directly on the `td` element

File: client/app/telecom/overTheBox/tasks/overTheBox-tasks.html [hunk 10]
```diff
@@ -32,7 +32,9 @@
                     <tbody data-ng-if="OverTheBoxTasks.loaders.init">
                         <tr>
                             <td colspan="3">
-                                <oui-loader data-size="m"></oui-loader>
+                                <div class="text-center">
```
Comment (Jisay) [inline client/app/telecom/overTheBox/tasks/overTheBox-tasks.html]: apply `text-center` class directly on the `td` element

**Pull request merged**